### ISSUE TITLE
Add Text Item - Add Formatting Options

### ISSUE THREAD

**Issue by roshanikothari** [tainacan/tainacan#395]
**Context and motivation**

We would like to add a Text item, but the text has links. Currently there is now way to add any links. 

**Ideal solution**

Add formatting options like the ones included when you add an issue in Github. 

**Alternative solutions**

Allow for Posts or Pages or Gutenberg Blocks to be added as Items in a Collection?

**Additional context**

For example, we are moving over our Resources from Zendesk. Here's the link to it.
https://plasticpollutioncoalition.zendesk.com/hc/en-us/articles/224373087-General-resources-for-all-grades
The first item in there has links in it.
Marine Debris Program, Educational Materials - NOAA,
the National Oceanic and Atmospheric Administration

**Comment by mateuswetah** [comment]
Hi @roshanikothari .

I am really sorry but we didn't saw your issue previously. I hope it is not too late to discuss this. 

In any case, @leogermani and @vnmedeiros this might be the kind of situation where that "post-as-item" discussion comes in.

**Comment by leogermani** [comment]
> Allow for Posts or Pages or Gutenberg Blocks to be added as Items in a Collection?

I love this idea. To have a fourth document type which is a WordPress post/page

**Comment by mateuswetah** [comment]
> I love this idea. To have a fourth document type which is a WordPress post/page

Yeah it does not seems so complicated to me... We can get the post content on the `get_document_as_html`... It would increase the potential of how items pages are made. The only tricky part, it seems, is that the content itself wont be fetch during search....

**Comment by leogermani** [comment]
> The only tricky part, it seems, is that the content itself wont be fetch during search....

There's a hidden document_index (I dont remember the name) metadata which we use to store the contents of an indexed PDF for search purposes. You could use the same to make the contents of the post searchable.

The fragile part is that the post can be deleted and the item can become orphan

**Comment by mateuswetah** [comment]
Well, the same would happen with media deleted from the media gallery, no?

**Comment by leogermani** [comment]
> Well, the same would happen with media deleted from the media gallery, no?

yes, true. Don't remember if there is any safeguard against that. There might be.

**Comment by vnmedeiros** [comment]
> yes, true. Don't remember if there is any safeguard against that. There might @leogermani 

in the case of PDF, on removing it from the main document of item also removes the content inserted in the metadata used for searching.
it could be something similar... but how to know that a "post" is used by an item?
